library/core/src/prelude/v1.rs

```diff
@@ -1,8 +1,6 @@
-//! The core prelude
+//! The first version of the core prelude.
 //!
-//! This module is intended for users of libcore which do not link to libstd as
-//! well. This module is imported by default when `#![no_std]` is used in the
-//! same manner as the standard library's prelude.
+//! See the [module-level documentation](../index.html) for more.
```

Comment:
```suggestion
//! See the [module-level documentation](super) for more.
```
This uses `//!`, so rustdoc resolves it relative to `v1`, not to `prelude`.